.github/workflows/node-ci.yml

```diff
@@ -166,6 +166,17 @@ jobs:
         run: |
           cmake --build build -j $(nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null)
 
+      - name: Run render tests on macOS
+        if: runner.os == 'macOS'
+        run: ./build/mbgl-render-test-runner --manifestPath metrics/macos-xcode11-release-style.json
+
+      - name: Upload render test artifacts
+        uses: actions/upload-artifact@v3
+        if: always()
```

Comment:
Not sure what `always()` does, but are you sure you want that on linux?

Reply:
This is a github actions instruction and works with all platforms
 It means that this step should run independent of the exit code of the previous step. If render tests fail, the exit code is non-zero but we always want to upload the artifacts...

Reply:
Ah your point is that on linux we don't have the artifact? Yeah that needs to be fixed...

Reply:
CI passed because the action issues only a warning if the artifact file is missing.

Reply:
I don't know how to tune the logic of the step such that it only runs on macos. But since it does not fail on linux, we can leave it here.